PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/runner/IntelliJFunctionContext.java

```diff
@@ -81,22 +82,16 @@ public IProject getProject() {
         return null;
     }
 
-    public Map<String, String> getTelemetryProperties(Map<String, String> properties) {
-        HashMap result = new HashMap();
-
+    public Map<String, String> getTelemetryProperties() {
+        final HashMap result = new HashMap();
         try {
-            if (properties != null) {
-                result.putAll(properties);
-            }
-            result.put("runtime", this.getRuntime().getOs());
+            result.put("runtime", this.getOs());
```

Comment:
runtime is not only os?

Reply:
function runtime including os and host java version, java version is recorded in following property